pyproject.toml

```diff
@@ -69,6 +69,7 @@ jpeg2000 = [
   "lxml>=4.9.0,!=5.0.0",
 ]
 map = [
+  "contourpy>=1.2.0"
```

Comment:
Does this need to be added?

Reply:
I guess not as I tried test ContourPy without ` "contourpy>=1.2.0"` and it worked fine

Reply:
The code (now) does a direct import from `contourpy`, so it needs to be added as an explicit dependency